tutorials/javaInstallationMac.md

```diff
@@ -0,0 +1,79 @@
+{% set title="Java 17 Installation Guide for Mac Users" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+## Introduction
+
+This guide provides detailed steps to install Java JDK-FX 17.0.11.fx-zulu on macOS using SDKMAN.
```

Comment:
```suggestion
This guide provides detailed steps to install Java JDK-FX 17.0.11.fx-zulu on macOS using SDKMAN.

SDKMAN is a tool that simplifies the process of installing, switching between, and managing multiple versions of software development kits on Unix-based systems. It's particularly useful for Java developers as it allows easy installation and management of different Java versions.
```

possible suggestion:
add more context about SDKMAN in the introduction. Many users, especially beginners, might not be familiar with SDKMAN. A brief explanation of what SDKMAN is and why it's beneficial for managing Java versions could help users understand the process better.

Reply:
I agree, @damithc can we add this in?

Reply:
@hjungwoo01 Yes, good suggestion. Go ahead.